vlsi/example-sky130.yml

```diff
@@ -0,0 +1,191 @@
+# Technology Setup
+# Technology used is Sky130
+vlsi.core.technology: sky130
+
+vlsi.core.max_threads: 12
+
+# Technology paths
+technology.sky130:
+  sky130_pdk: "path-to-skywater-pdk/"
+  sky130_nda: "path-to-skywater-src-nda/"
+  sky130A: "path-to-sky130A/"
+  openram_lib: "path-to-sky130_sram_macros/" 
+
+# Mentor environment variables
+mentor.extra_env_vars:
+  - PDK_HOME: "path-to-skywater-src-nda/s8/V2.0.1"
```

Comment:
Can this be placed in Sky130's `defaults.yml` to reduce the number of keys users need to set? It should be:

```
mentor.extra_env_vars_meta: lazydeepsubst
mentor.extra_env_vars:
  - PDK_HOME: "${technology.sky130.sky130_nda}/s8/V2.0.1"
```